pkg/lib/configreader/string.go

```diff
@@ -259,6 +260,12 @@ func ValidateStringVal(val string, v *StringValidation) error {
 		}
 	}
 
+	if v.DockerImage {
```

Comment:
Since this validation allows empty strings, we have two options for how to make that clear:

1. We can rename the validation to `v.DockerImageOrEmpty`
2. We can convert the struct field to be a pointer (since these validations aren't evaluated on `nil`)

Which do you think is better? I'd be happy to chat about this

Reply:
I suggest renaming it to `v.DockerImageOrEmpty` because I see another field following the same pattern: `AlphaNumericDashDotUnderscoreOrEmpty`. What do you say?

Reply:
Sounds good!